docs/spec/staking/state.md

```diff
@@ -2,46 +2,46 @@
 
 ### Pool
 
-The pool is a space for all dynamic global state of the Cosmos Hub.  It tracks
-information about the total amounts of Atoms in all states, moving Atom
-inflation information, etc.
+The pool dynamically tracks information about the total amounts of staking
+tokens and their states (bonded or loose). Note that `NotBondedTokens` _does_
+include both `unbonding` tokens, and fully `unbonded` tokens. 
 
  - Pool: `0x01 -> amino(pool)`
 
 ```golang
 type Pool struct {
-    LooseTokens         sdk.Int   // tokens not associated with any bonded validator
-    BondedTokens        sdk.Int   // reserve of bonded tokens
+    NotBondedTokens sdk.Int   // tokens not associated with any bonded validator
+    BondedTokens    sdk.Int   // reserve of bonded tokens
 }
 ```
 
 ### Params
 
 Params is global data structure that stores system parameters and defines
-overall functioning of the staking module. 
+overall functioning of the staking module. it is stored 
 
- - Params: `0x00 -> amino(params)`
+ - Params: `Paramsspace("staking") -> amino(params)`
 
 ```golang
 type Params struct {
-    MaxValidators uint16 // maximum number of validators
-    BondDenom     string // bondable coin denomination
+	UnbondingTime time.Duration // time duration of unbonding
+    MaxValidators uint16        // maximum number of validators
+    BondDenom     string        // bondable coin denomination
 }
 ```
 
 ### Validator
 
 Validators are identified according to the `OperatorAddr`, an SDK validator
 address for the operator of the validator.
-
 Validators also have a `ConsPubKey`, the public key of the validator used in
 Tendermint consensus. The validator can be retrieved from it's `ConsPubKey`
 once it can be converted into the corresponding `ConsAddr`. Validators are
 indexed in the store using the following maps:
 
-- Validators: `0x02 | OperatorAddr -> amino(validator)`
-- ValidatorsByConsAddr: `0x03 | ConsAddr -> OperatorAddr`
-- ValidatorsByPower: `0x05 | power | blockHeight | blockTx  -> OperatorAddr`
+- Validators: `0x21 | OperatorAddr -> amino(validator)`
```

Comment:
Again, more explanation here would be great.

Reply:
see my previous comment on byte structure, also the following descriptions are quite comprehensive on describing the usage of the indexes